### PR TITLE
Add runn to test SSE job with a sample which returns non-zero exit code.

### PR DESCRIPTION
**Pull request by SECuchida**
The python program exits with non-zero code.
When this case, the job fails and the python log is copied to engine log.

### CODE REVIEW

File: test/tools/sse/sample_circuit_nonzero_exit.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import skqulacs
+import pyqubo
+import openjij
+import cirq
+import pennylane
+import openfermion
+import time
+
+# raise exception
+import unknown_module
```
Comment (SECuchida) [inline test/tools/sse/sample_circuit_nonzero_exit.py:21]: Here the program exits with non-zero code.

**Comment by massn** [comment]
Confirmed it works. 
LGTM!
Please merge.

**Comment by SECuchida** [comment]
Thank you!
